src/WriteIndexListsMain.cpp

```diff
@@ -87,7 +87,8 @@ int main(int argc, char** argv) {
     index.dumpAsciiLists(lists, decodeGapsAndFrequency);
 
     Engine engine;
-    QueryExecutionContext qec(index, engine);
+    SubtreeCache cache(NOF_SUBTREES_TO_CACHE);
```

Comment:
As I'm not that familiar with the codebase, is using the cache here beneficial? Or is this more or less a no-op?

Reply:
This is just moving the cache out of the `QueryExecutionContext` while still using it anywhere it was used before

Reply:
Fine by me.